templates/webapi/test/link_context.html.ep

```diff
@@ -0,0 +1,15 @@
+% layout 'bootstrap';
+% content_for 'head' => begin
+  %= asset 'codemirror.js'
+  %= asset 'codemirror.css'
+% end
+
+<div class="container">
+        <p>
+            File Path:
+                <code><%= $contextpath %></code>
```

Comment:
The only difference between this file and `src.html.ep` is "File Path" (which should likely better be "File path:") and that `src.html.ep` has a link. I don't think these 2 differences justify having this whole template twice.

Reply:
I guess the comment from 19 days ago is still valid.